frontend/src/components/ViewActivities/editActivity.js

```diff
@@ -54,7 +39,6 @@ class EditActivity extends React.Component {
   
   /**
    * Edit an activity in the database upon form submission.
-   * TODO: Update times as well! This only does the text field forms (#64).
    */
   editActivity() {
```

Comment:
The diff isn't showing it, but this function (which was completely redone) is the single biggest and most important change from this pull request.